boards/riscv32/arty_bm310/doc/arty_bm310.rst

```diff
@@ -0,0 +1,65 @@
+.. _arty_bm310:
+
+CloudBEAR BM-310
+##############
```

Comment:
Title underline needs to be at least as long as the title, so add a couple more ``##``'s